app/controllers/distributions_controller.rb

```diff
@@ -101,7 +101,10 @@ def create
       respond_to do |format|
         format.turbo_stream do
           flash.now[:error] = flash_error
-          render turbo_stream: turbo_stream.replace("flash", partial: "shared/flash")
+          render turbo_stream: [
+            turbo_stream.replace(@distribution, partial: "form", locals: { distribution: @distribution }),
```

Comment:
This is needed to replace the distribution form's distribution record that contains the errors. This line of code makes it so the errors now appear beneath the input fields with issues.